lib/ImportStatements.js

```diff
@@ -240,7 +240,7 @@ export default class ImportStatements {
       importsArray,
       (importStatement: ImportStatement): boolean =>
         !importStatement.isParsedAndUntouched(),
-    );
+    ).reverse();
```

Comment:
Nice! I saw you made a comment about this elsewhere. Although I would prefer for this to be either a separate commit in the PR or a separate PR altogether. So that we're not mixing concerns here.